packages/stark-core/tslint.json

```diff
@@ -1,6 +1,30 @@
 {
   "extends": ["tslint:latest", "tslint-sonarts", "codelyzer", "tslint-config-prettier", "../stark-build/config/tslint.json"],
   "rules": {
+    "use-view-encapsulation": false,
```

Comment:
If we disable this rule in stark-core, stark-ui, stark-starter and stark-showcase, why not disable it directly in the tslint.json file in stark-build to apply this rule everywhere?